packages/vscode-extension/src/utilities/packageManager.ts

```diff
@@ -28,6 +36,31 @@ export async function resolvePackageManager(): Promise<PackageManagerInfo | unde
   const workspacePath = findWorkspace(appRootPath);
 
   async function findPackageManager(workspace: string) {
+    const { packageManager } = getLaunchConfiguration();
+
+    if (packageManager) {
+      if (!isPackageManager(packageManager)) {
+        Logger.warn(
+          `Package manager provided in launch configuration: ${packageManager} is not supported by radon IDE`
+        );
+        return;
+      }
+      return packageManager;
+    }
+
+    try {
+      const manager = require(path.join(workspace, "package.json")).packageManager;
```

Comment:
we should use require with no-cache option or just read the file. Using require will cache the package for as long as the extension process exists